### ISSUE TITLE
Cognitive function test definition relies on informative note

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag#1343]
The cognative function test definition has an exception for remembering name, e-mail and phone number. This exception should not be put in a note, as notes in WCAG are not normative. Instead of using a note, this should be put as an explicit exception in the definition.

> Remembering your own name, email address, or phone number is not considered a cognitive function test.
> Source: https://www.w3.org/TR/WCAG22/#dfn-cognitive-function-test

**Comment by gradualclearing** [comment]
I have created pull request #1365

**Comment by alastc** [comment]
Noting this duplicates #1310

**Comment by alastc** [comment]
Closing based on the changes here:
https://github.com/w3c/wcag/pull/1411/files

Minutes: https://www.w3.org/2020/09/22-ag-minutes.html#item03
